examples/flutter_gallery/ios/Runner/Info.plist

```diff
@@ -43,5 +43,9 @@
 	<false/>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSCameraUsageDescription</key>
+  <string>Can we use the camera to scan barcodes?</string>
```

Comment:
This looks like (accidental) inconsistent spacing. All other lines use tabs, except for these 3.